admin_setting.go

```diff
@@ -0,0 +1,461 @@
+package tfe
+
+import (
+	"context"
+)
+
+// Compile-time proof of interface implementation.
+var _ AdminSettings = (*adminSettings)(nil)
+
+// AdminSettings describes all the admin settings related methods that the Terraform Enterprise API supports.
+// Note that admin settings are only available in Terraform Enterprise.
+//
+// TFE API docs: https://www.terraform.io/docs/cloud/api/admin/settings.html
+type AdminSettings interface {
+	// GetGeneral returns the general settings
+	GetGeneral(ctx context.Context) (*AdminGeneralSetting, error)
```

Comment:
I think `Read*` for getter funcs might be a little more consistent with the rest of the API. In spelunking history for one of your other PRs I actually noted this was changed from `Fetch` intentionally, too!